docs/developer/payments/transactions.mdx

```diff
@@ -367,15 +367,20 @@ sequenceDiagram
 
 Follow [Transactions Webhook Events](/developer/extending/webhooks/synchronous-events/transaction.mdx) guide.
 
-### Automatic checkout completion
+### Automatic Checkout completion
 
-The checkout process can be automatically completed based on the transaction events, once the full payment is made.
+The `Checkout` can be automatically converted into an [`Order`](/api-reference/orders/objects/order.mdx), once the related [`TransactionItems`](/api-reference/payments/objects/transaction-item.mdx) fully cover `Checkout`.
 
-A checkout is considered fully paid when the [`Checkout.authorizeStatus`](/api-reference/checkout/objects/checkout#checkoutchargestatuscheckoutchargestatusenum---) 
+:::caution
+This feature isn't compatible with [legacy Payments API](/developer/extending/plugins/payment-gateways.mdx) that uses [plugins](/developer/extending/plugins/overview.mdx) (instead of apps). For example this setting won't work with [Stripe plugin](/developer/app-store/plugins/stripe).
```

Comment:
Nothing will happen, it doesn't trigger automatic checkout completion. This entire setting doesn't make sense at all with legacy Payment API, since you actually can't even make payment with it without calling `checkoutComplete`: it works by providing parameters in `checkoutComplete` mutation, then if payment is successful you **immediately** create an Order.